docs/catalog/sinks/discord.rst

```diff
@@ -0,0 +1,51 @@
+Discord
+#################
+
+Robusta can send playbook results to Discord.
+
+To configure the Discord sink we will need the Discord webhook url
+
+.. note::
+
+    2-way interactivity (``CallbackBlock``) isn't implemented yet.
+
+Get your discord webhook url
+------------------------------------------------
+
+1. Open the Discord channel you want to receive Robusta event notifications.
+2. From the channel menu, select Edit channel.
+
+    .. image:: /images/discord_edit_channel.png
+      :width: 400
+      :align: center
+
+3. Select Integrations.
+4. If there are no existing webhooks, select Create Webhook. Otherwise, select View Webhooks then New Webhook.
+
+    .. image:: /images/discord_create_webhook.png
+      :width: 1000
+      :align: center
+
+5. Optional. You can enter the name of the bot and the avatar to post the message, but Robusta will change them if allowed.
```

Comment:
Do we really enable that?
I didn't see a way to configure it in the sink.
And when sending the message, the name seems to always be `Robusta`